test/index.js

```diff
@@ -2134,25 +2220,21 @@ describe('wait()', () => {
 
     it('delays for timeout ms', async () => {
 
-        // DO NOT USE Hoek.Bench() to time this, as its test depends on Hoek.wait()
-
-        const start = process.hrtime.bigint();
+        let timeout = false;
+        setTimeout(() => (timeout = true), 10);
         await Hoek.wait(10);
-        const end = process.hrtime.bigint();
 
-        expect(end - start).to.be.between(10_000_000, 20_000_000);
+        expect(timeout).to.be.true();
```

Comment:
This confirms that the timer takes at least 10 scheduled ms, but doesn't place an upper bound the way the old test did, as imprecise as it may have been.  If we're going to tackle this now then perhaps we might as well use the same technique to test the other bound.  Something like:
```js
let timeoutBefore = false;
let timeoutAfter = false;

setTimeout(() => (timeoutBefore = true), 10);
const wait = Hoek.wait(10);
setTimeout(() => (timeoutAfter = true), 10);

await wait;

expect(timeoutBefore).to.be.true();
expect(timeoutAfter).to.be.false();
```

Reply:
Good idea. I have updated the tests.